src/annotator/test/features-test.js

```diff
@@ -51,10 +56,10 @@ describe('features - annotation layer', function() {
     });
 
     it('should warn when accessing unknown flags', function() {
-      assert.notCalled(console.warn);
+      assert.notCalled(fakeWarnOnce);
       assert.isFalse(features.flagEnabled('unknown_feature'));
-      assert.calledOnce(console.warn);
-      assert.calledWith(console.warn, 'looked up unknown feature');
+      assert.calledOnce(fakeWarnOnce);
```

Comment:
Perhaps use `calledOnceWith` here?

Reply:
Oh wait ... it turns out there is no `calledOnceWith` function. That only exists in Python-land (as `called_once_with`)

Reply:
Are we not using the chai plugin for `sinon`? https://www.chaijs.com/plugins/sinon-chai/

Reply:
Ah, we're just using Sinon's [`assert.expose` function](https://sinonjs.org/releases/v7.3.2/assertions/) -  see https://github.com/robertknight/client/blob/ac91921b940ce23bf5b1981de8482f2471aff3ef/src/sidebar/test/bootstrap.js#L4

Depending on what else that plugin gives us, maybe we should consider it.